_pages/how-we-work/leading-projects.md

```diff
@@ -17,6 +17,7 @@ Here is some of what _success_ includes:
 * Establishing a healthy team with positive working relationships
 * The strategic direction of the project (including alignment with a portfolio, when relevant)
 * Managing work so that team members can be fully billable
+* Establishing a system to fairly rotate administrative or clerical tasks between teammates
```

Comment:
this is minor, but to be a bit less stiff and wordy, what do you think of trimming down to "Rotating administrative and clerical tasks among teammates"?

Reply:
Hmm, in this case I want to emphasize that the goal is to establish systems up front rather than assign tasks week-by-week.